core/scripts/chaincli/config/config.go

```diff
@@ -33,14 +37,18 @@ type Config struct {
 	GasCeilingMultiplier uint16 `mapstructure:"GAS_CEILING_MULTIPLIER"`
 	MinUpkeepSpend       int64  `mapstructure:"MIN_UPKEEP_SPEND"`
 	MaxPerformGas        uint32 `mapstructure:"MAX_PERFORM_GAS"`
+	MaxCheckDataSize     uint32 `mapstructure:"MAX_CHECK_DATA_SIZE"`
+	MaxPerformDataSize   uint32 `mapstructure:"MAX_PERFORM_DATA_SIZE"`
 	FallbackGasPrice     int64  `mapstructure:"FALLBACK_GAS_PRICE"`
 	FallbackLinkPrice    int64  `mapstructure:"FALLBACK_LINK_PRICE"`
 	Transcoder           string `mapstructure:"TRANSCODER"`
 	Registrar            string `mapstructure:"REGISTRAR"`
 
-	// Keepers Config
+	// Upkeep Config
+	OCR2Keepers                     bool                   `mapstructure:"KEEPER_OCR2"`
```

Comment:
nit. move to line 28, with OCR Config params?

Reply:
Done